tests/_utils.py

```diff
@@ -519,4 +523,4 @@ def pandas_equals(df1: Union[pd.DataFrame, pd.Series], df2: Union[pd.DataFrame,
 
 def assert_pandas_equals(df1: Union[pd.DataFrame, pd.Series], df2: Union[pd.DataFrame, pd.Series]) -> None:
     df1, df2 = to_pandas(df1), to_pandas(df2)
-    assert_equal(df1, df2)
```

Comment:
Is this going to work for Pandas Series as well? The advantage of using `assert_equal` is that it automatically dispatches to the appropriate assertion. See [here](https://github.com/pandas-dev/pandas/blob/main/pandas/_testing/asserters.py#L1227).

Reply:
Unfortunately it is deprecated in 2.x. I've updated to use `assert_frame_equal` & `assert_series_equal`.